src/Confluent.Kafka/Deserializers.cs

```diff
@@ -0,0 +1,279 @@
+using System;
+using System.Text;
+
+
+namespace Confluent.Kafka
+{
+    /// <summary>
+    ///     Deserializers that can be used with <see cref="Confluent.Kafka.Consumer{TKey, TValue}" />.
+    /// </summary>
+    public static class Deserializers
+    {
+        /// <summary>
+        ///     Deserializes a UTF8 encoded string.
+        /// </summary>
+        public static Deserializer<string> UTF8 = (topic, data, isNull) =>
+        {
+            if (isNull)
+            {
+                return null;
+            }
+
+            #if NETCOREAPP2_1
+                return Encoding.UTF8.GetString(data);
+            #else
+                return Encoding.UTF8.GetString(data.ToArray());
+            #endif
+        };
+
+        /// <summary>
+        ///     Deserializes a null value to a null value.
+        /// </summary>
+        public static Deserializer<Null> Null = (topic, data, isNull) =>
+        {
+            if (!isNull)
+            {
+                throw new System.ArgumentException("Deserializer<Null> may only be used to deserialize data that is null.");
```

Comment:
As the argument isn't passed by the user but by a consumed message I think perhaps ValueException (or whatever) might be better

Reply:
Agree this is not good, but I think it's the best of the standard exception types and probably don't want to invent another one.

https://stackoverflow.com/questions/774104/what-exceptions-should-be-thrown-for-invalid-or-unexpected-parameters-in-net